From e5bc72cf68d3556530a5add7f1426733df05e044 Mon Sep 17 00:00:00 2001
From: Ethan Chen <intervigil@gmail.com>
Date: Sun, 20 May 2018 16:58:01 -0700
Subject: [PATCH 10/11] build: Use TOPDIR, not TOP

* HTTP and RRO rules are not being loaded correctly due to the bad path
  variable usage.

Change-Id: I5b2677c22aa878e46e01f4e5b31769e21937c38a
---
 build/core/config.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/core/config.mk b/build/core/config.mk
index 887a6220..8abf7050 100644
--- a/build/core/config.mk
+++ b/build/core/config.mk
@@ -20,6 +20,6 @@ FRAMEWORK_LINEAGE_PLATFORM_API_FILE := $(TOPDIR)lineage-sdk/api/lineage_current.
 FRAMEWORK_LINEAGE_PLATFORM_REMOVED_API_FILE := $(TOPDIR)lineage-sdk/api/lineage_removed.txt
 FRAMEWORK_LINEAGE_API_NEEDS_UPDATE_TEXT := $(TOPDIR)vendor/lineage/build/core/apicheck_msg_current.txt
 
-BUILD_HTTP_PREBUILT := $(TOP)/vendor/lineage/build/core/http_prebuilt.mk
+BUILD_HTTP_PREBUILT := $(TOPDIR)vendor/lineage/build/core/http_prebuilt.mk
 
-BUILD_RRO_SYSTEM_PACKAGE := $(TOP)/vendor/lineage/build/core/system_rro.mk
+BUILD_RRO_SYSTEM_PACKAGE := $(TOPDIR)vendor/lineage/build/core/system_rro.mk
-- 
2.17.0

